kvirt/cluster/k3s/__init__.py

```diff
@@ -36,10 +36,12 @@ def update_ip_alias(config, nodes):
 def scale(config, plandir, cluster, overrides):
     plan = cluster
     data = {'cluster': cluster, 'kube': cluster, 'kubetype': 'k3s', 'image': 'ubuntu2004', 'sdn': 'flannel',
-            'extra_scripts': [], 'cloud_lb': True, 'cloud_native': False, 'ctlplanes': 1, 'workers': 0}
+            'extra_scripts': [], 'cloud_lb': None, 'cloud_native': False, 'ctlplanes': 1, 'workers': 0}
```

Comment:
cloud_lb should default to True

Reply:
Can you elaborate more on why. E.g. cloud_lb is not used on-prem. So setting it to `True` won't it fault certain things if one uses `kcli scale` on-prem?

Reply:
cloud_lb is only checked when platform is cloud

Reply:
Where does the check take place. Can't find it.

Reply:
https://github.com/karmab/kcli/blob/f31deb679c11f578964750fa75fd5216f9cedbfe/kvirt/cluster/k3s/__init__.py#L215

Reply:
But, that check is in the create() not in the scale() in k3s/__init__.py ... so I still think that cloud_lb should be updated as done for the scale() in this PR.

Reply:
I think cloud_lb is not used at all in the scale process

Reply:
If it isn't then let's completely remove it from the func. definition. Agree? Then we indeed not need the extraneous code in the scale() in k3s init py.